omega/omega-transport/omega-transport-servicecomb/src/main/java/org/apache/servicecomb/saga/omega/transport/servicecomb/SagaConsumerHandler.java

```diff
@@ -54,6 +62,8 @@ public void handle(Invocation invocation, AsyncResponse asyncResponse) throws Ex
           omegaContext.globalTxId(),
           LOCAL_TX_ID_KEY,
           omegaContext.localTxId());
+    } else {
+      LOG.warn("The OmegaContext is not injected, The SagaConsumerHandler is disabled.");
```

Comment:
The SagaConsumerHandler just skip the work,  the log level can be info.

Reply:
Done